src/ai-assistant/src/tools/layer-utils.ts

```diff
@@ -0,0 +1,180 @@
+import {ColorMap, ColorRange, ColorLegends, COLOR_RANGES} from '@kepler.gl/constants';
```

Comment:
Can these files be more thoughtfully named? This isn't just a another dump of utils is it? These are exposed function to let the assistant create layers?